src/sslyze_csv2mongo.py

```diff
@@ -180,9 +181,9 @@ def store_data(clean_federal, agency_dict, db_config_file):
         date_items = ("Not Before", "Not After")
         for date_item in date_items:
             if row[date_item]:
-                row[date_item] = datetime.datetime.strptime(
+                row[date_item] = datetime.strptime(
                     row[date_item], "%Y-%m-%dT%H:%M:%S"
-                ).replace(tzinfo=timezone("US/Eastern"))
+                ).replace(tzinfo=ZoneInfo("US/Eastern"))
```

Comment:
The `US/Eastern` time zone is dependent on the underlying `tzdata` OS package being installed [here](https://github.com/docker-library/python/blob/5906158d277460e04837c45f5b22051bbf28af92/3.12/alpine3.19/Dockerfile#L20). This is because it requires the `backwards` file that links `Us/Eastern` to `America/New_York` which apparently was [changed in 1995](https://github.com/eggert/tz/blob/d4d3c3ba2361c413b765ac02c7ffea6d4ce44152/NEWS#L5903-L5911).

```suggestion
                ).replace(tzinfo=ZoneInfo("America/New_York")
```

Reply:
@jsf9k I made a syntax error in my suggestion by accidentally dropping a parentheses. I apologize 😞